src/components/form/components/select.js

```diff
@@ -21,30 +21,36 @@ const Select = ({
   name,
   domRef,
   ...props
-}) => (
-  <Element
-    domRef={domRef}
-    className={classnames('select', className, {
-      [`is-${size}`]: size,
-      [`is-${color}`]: color,
-      'is-loading': loading,
-      'is-multiple': multiple,
-    })}
-    style={style}
-  >
+}) => {
+  function defaultValue(isMultiple) {
+    return isMultiple ? [] : '';
+  }
```

Comment:
the parameter `isMultiple` is unnecessary since the `multiple` prop is in scope. also please add a jsdoc comment (`/** */`) above the function to describe what it does.